nsenter/nsenter_test.go

```diff
@@ -1,91 +0,0 @@
-package nsenter
-
-import (
-	"encoding/json"
-	"fmt"
-	"os"
-	"os/exec"
-	"strings"
-	"testing"
-)
-
-type pid struct {
-	Pid int `json:"Pid"`
-}
-
-func TestNsenterAlivePid(t *testing.T) {
```

Comment:
Sorry i forgot to restore it when i changed the env. It was deleted because it has to be rewritten to test for invalid namespaces path, not invalid pids anymore since we only care about the namespace paths now.

I will restore it with new environment variables.